### PR TITLE
Add protobuf license needed for internal use.

### PR DESCRIPTION
**Pull request by hsubox76**
See cl/358448976 (internal).

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -200,3 +202,39 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+------------------------------------------------------------------------------
+
+For Google protobuf source.
+
+file: src/packages/firestore/src/protos/google/protobuf/any.proto
```
Comment (Feiyang1) [inline LICENSE:210]: What should the path here?

Comment (hsubox76) [reply LICENSE:210]: It's a relative path in google3, it's copied verbatim from the CL.

Comment (Feiyang1) [reply LICENSE:210]: But this license is for the public version of the SDK, should it use the path in Github?

Comment (hsubox76) [reply LICENSE]: Unless we want to write a special script to reformat it on google3 import, it's probably best to make it applicable to wherever it is. I tried 
```
[source code root]/packages/firestore/src/protos/google/protobuf/any.proto
```

**Review by Feiyang1 — APPROVED** [review]
Do we need to update a script to change the path for google3?

**Comment by hsubox76** [comment]
Discussed with CL author, the path does not need to be strict, it is just for providing context to any human readers of the license.

**Pull request merged**